src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -116,4 +120,96 @@ public static boolean isValidClassicAddress(String address) {
   public static String toTransactionHash(String transactionBlobHex) {
     return javaScriptUtils.toTransactionHash(transactionBlobHex);
   }
+
+  /**
+   * Convert from units in drops to XRP.
+   *
+   * @param drops An amount of XRP expressed in units of drops.
+   * @return A String representing the drops amount in units of XRP.
+   * @throws IllegalArgumentException if drops is in an invalid format.
+   */
+  public static String dropsToXrp(String drops) throws IllegalArgumentException {
+    Preconditions.checkNotNull(drops);
+
+    String dropsRegex = "^-?[0-9]*['.']?[0-9]*$";
+    Pattern dropsPattern = Pattern.compile(dropsRegex);
+    Matcher dropsMatcher = dropsPattern.matcher(drops);
+
+    if (!dropsMatcher.matches()) {
+      throw new IllegalArgumentException(String.format(
```

Comment:
Non-blocking: 

I had used [`INVALID_INPUTS`](https://github.com/xpring-eng/xpring4j/blob/master/src/main/java/io/xpring/xrpl/XRPExceptionType.java#L5) and `XRPException` to handle cases like this in the past. 

@nhartner what's more idiomatic here? Always to throw the most specific exception? I can revise existing code if so.

Reply:
Maybe I should also be using `XRPException` here because I use the Swift equivalent in the Swift analogue to this PR... probably because Swift doesn't have the same palate of built-in error types like Java does

Reply:
went this route